runtime/near-vm-runner-standalone/src/script.rs

```diff
@@ -81,7 +82,7 @@ impl Script {
 
     pub(crate) fn vm_config_from_file(&mut self, path: &Path) {
         let data = fs::read(path).unwrap();
-        let vm_config = serde_json::from_slice(&data).unwrap();
+        let vm_config = serde_json::from_slice::<DeserializedWithDebug<_>>(&data).unwrap().0;
```

Comment:
Would it help to keep the code more straightforward if some sort of a helper like below was added? 

```
pub fn near_primitives::serialize::from_json_slice<T>(data: &[u8]) -> Result<T, ...> {
    serde_json::from_slice::<DeserializedWithDebug<T>>(data)
}
```

That way the straightforward thing to solve the problem would also by default do the right thing (introduce various log hooks) without anybody needing to go out of their way to opt-in.